### DISCUSSION TITLE
Presentations

### DISCUSSION THREAD

**Discussion by sciabarracom** [nuvolaris/nuvolaris#7] [General]
Hi All I am Michele Sciabarrà!

I am the founder of the Nuvolaris prject.

I am member of the Apache OpenWhisk project (author of the runtime proxy actionloop used by many runtimes), author of  the book Learning Apache OpenWhisk published by O'Reilly, formerly Nimbella Europe Manager and (briefly) Developer Advocate at Digital Ocean who acquired Nimbella.

This project aims to build an open source, production-ready, distribution of Apache OpenWhisk that installs in every kubernetes, works flawless and it is easy to use with good development tools.

If you ware interested to collaborate, please get in touch and introduce yourself.

**Comment by saniales** [comment]
Hello there,
 
My name is Alessandro Sanino and I am a cloud architect and blockchain researcher. Although I have not full experience in System configurations of K8s clusters, I am more than happy to help doing other tasks, like QA, code reviews

Hope to be of help @sciabarracom 

DAJE

**Reply by sciabarracom** [reply]
Welcome Alessandro!

**Reply by AmbraDanesin** [reply]
Hello Alessandro and welcome here!
We as a time are happy to have you and... Ao, ho visto ora il "daje" finale, che sto a parla come il big boss inglese a fare.
Bella Ale, ben arrivato, è un piacere averti con noi!

**Comment by mauradona** [comment]
Hi All,

my name is Mauro (aka Mauradona). I have been a software engineer for more than 20 years. Currently I'm working on nodejs to solve the company problems where I work :(
I would like to create some of exceptional usefull tool with the last information technologies so I'm here.

I hope to collaborate with you.
              Mauro

**Reply by sciabarracom** [reply]
Welcome Mauro!

**Reply by AmbraDanesin** [reply]
That's great, Mauro! (...and yes, your nick "Mauradona" is like my favorite thing for at least today)
I'm sure you'll be able to do something great, good luck!

**Comment by eduardomecchiafernandez** [comment]
Hi all!

My name's Eduardo Mecchia, and I'm currently working on a project as a front-end developer for Accenture Interactive, but I've also studied back-end technologies. I'm studying containerization with Docker and K8s and, judging by the sheer level of organization, I think this project will go far. so I hope to help wherever's possible.

**Reply by sciabarracom** [reply]
That would be great I think there is a lot in the project you can be interested in.

**Reply by AmbraDanesin** [reply]
Hi Eduardo!
Thank you, the fact that you (and others!) believe in us makes us stronger!

**Comment by asclabs** [comment]
Hello to everyone,

My name is Antonino S. Cutrì, i'm long time Software Architect with more of 20 years of experience,  and a Java developer since JDK 1.0, i will be happy to help with Spring  Cloud Function integration if needed.

**Reply by sciabarracom** [reply]
Hi Antonino!

**Reply by AmbraDanesin** [reply]
Everything is needed so don't worry about it, even just your presence is a great start 🌟

**Comment by dilox** [comment]
Hello Michele,
I always followed you in all kinds of crazy-super tech projects and this looks like the right one to have a lot of fun. Hope I can help

**Reply by sciabarracom** [reply]
You welcome Pierluigi

**Reply by AmbraDanesin** [reply]
So it seems that @sciabarracom has a fan here 👀

**Reply by sciabarracom** [reply]
Kinda. We have been working together in the latest 20 years...

**Reply by AmbraDanesin** [reply]
Not bad! At least he knows you very well eheh

**Comment by ferndem** [comment]
Hello to everyone,

My name is Fernando, I am a Software Architect with experience in .NET, JAVA and K8. I will be happy to help you in this super project.

**Reply by sciabarracom** [reply]
Welcome Fernando!

**Reply by AmbraDanesin** [reply]
Welcome Fernando! Thanks for the help e for the iper effective "super project". This sure will be one 😎

**Comment by ant4j** [comment]
Hello Michele and everyone,

my name is Antonio and I have 4 years of work experience in web applications development (Java Enterprise, last 1 year and a half on backend side, over my experience I dealt with the controller side of web clients and I had the opportunity to code in JavaScript also).
In my last experience I worked with Spring Boot, RESTful API and Mockito (for unit tests) for developing microservices.
I'm the one who followed your nimbella serverless training in italian started last january (thank you again) and from that I transformed an idea into a serverless web application with svelte, node and redis (and also a perl script).

It would be a pleasure to contribute in this project if It is possible.

Regards.

**Reply by sciabarracom** [reply]
Super happy to meet you... Sprng Boot is one of our targets

**Reply by AmbraDanesin** [reply]
Hi Antonio!
Great presentation and thank you for the interest, we will be happy to work together!

**Comment by mobs75** [comment]
Hello,
my name is Alessio, i' am a Software Architect with experience in Scala and Java, in recent years I have worked a lot in the world of distributed applications and cloud computing. 
I am very happy to give my contribution in this project.

**Reply by sciabarracom** [reply]
Welcome ALessio

**Reply by mobs75** [reply]
Grazie

**Reply by AmbraDanesin** [reply]
Ehi, Alessio! Thank you and I hope we will have the possibility to know each other! ❤️

**Comment by carloocchiena** [comment]
Hello guys

I'm Carlo, I am following Michele since a year and I like his approach and ideas. I'll be pleased to contribuite to the project.

I am fluent in .py and .ipynb, but in general I'd like to get my hands dirty, so I'll be digging issues and help whenever I can 

let's go!

**Reply by sciabarracom** [reply]
Hi Carlo! Very happy to see you here!

**Reply by AmbraDanesin** [reply]
And another fan for @sciabarracom! 👀
Hi Carlo, it's a pleasure, feel free to contact us when you want!

**Comment by mstocchi** [comment]
Hi all, 

My name is Manolo Stocchi, I'm a Software and Solution Architect and I've been working in the field for 20 years.
I'll be pleased to contribute in this very interesting project wherever is needed, from Java/Node/Go developments, K8s and cloud integrations, distributed solutions, devops tooling and so on.

**Reply by sciabarracom** [reply]
Welcome Manolo. I will publish soon a roadmap

**Reply by AmbraDanesin** [reply]
Hi Manolo! Where are you from? Your name suggest me Sud Italy but you last name maybe... Nord est?
Anyway, it's a pleasure, see you soon around!

**Comment by andrea78novi** [comment]
Hi guys,

I'm Andrea and I've been a Web Developer for 16 years.
I worked on several projects with different technologies but my skills are "sharper" if I can dedicate on front-end development.
Angular and Typescript my favourite stack.
I hope I can contribute in some way to the realization of this project.

**Reply by sciabarracom** [reply]
You welcome Andrea.

**Reply by AmbraDanesin** [reply]
The more the merrier, Andrea!
Welcome!

**Comment by halfpass25** [comment]
Hi Michele,

this is Pasquale Alfano writing.
Although the specific subject is outside my professional skills I'm glad to help spreading the new on this interesting project and provide support for what I can.
Unfortunately I only code in PHP/MySQL but my extensive technical background could be useful to spot critical areas or write docs.
Anyhow, I'm definitely able and willing to help giving birth to Nuvolaris!

"SI-PUO'-FAAAAAREEEE" (Frankenstein Jr. and many others....)

**Reply by sciabarracom** [reply]
You welcome Pasquale

**Reply by AmbraDanesin** [reply]
Si può fare sicuramente, Pasquale! Thank you for join us!

**Comment by aviola** [comment]
Hello guys,

My name is Alessio and I'm a software developer working on microservices using Java, C/C++, Docker and Kubernetes. I'm interested in learning about Go and cloud-native technologies in general.

I'll be happy to help you in this great project 🙂

**Reply by sciabarracom** [reply]
You welcome. And just today I was thinking that probably our cli tool will have to be done in Go as it will extend `wsk`  and other tools already written in Go.

**Reply by AmbraDanesin** [reply]
Hi, Alessio!
So far, how you're doing? Do you like the project?

**Comment by f4b3r** [comment]


**Reply by sciabarracom** [reply]
You welcpme Fabrizio.

**Reply by AmbraDanesin** [reply]
"Challenging" is the right word, Fabrizio! This is why it's so fun!

**Comment by ermalaliraj** [comment]
Hello everyone, my name is Ermal.

I am a Senior Developer with 13 years of experiences.
I have experience in back-end: Java, Spring Boot, RESful API, Microservices, Unit/Integration testing, Docker, Python.
And also in front-end: Javascript, AngularJS, Angular9, Automation Testing with Cucumber and Selenium.
Enthusiast in: DevOps, Cloud native, Kubernetes, Security, Angular, React.

I would be happy to give my contribution in any of the areas listed above for this interesting project.

**Reply by sciabarracom** [reply]
You welcome Ermal

**Reply by AmbraDanesin** [reply]
And we are happy to have here with us!
I'm sure that your presence will do a huge difference!

**Comment by claudiomantuano** [comment]
Hi all, 
I am Claudio and I have been working with Java and Spring since many years ago. Part of my job is to deal with microservices, REST, Kafka streams and testing through Mockito and TestContainer.
I would be happy to give my contribution if needed.
Claudio

**Reply by sciabarracom** [reply]
Hi Claudio welcome.

**Reply by AmbraDanesin** [reply]
It will be needed! So be ready, Claudio 🌟

**Comment by attiliocaravelli** [comment]
Hi anyone,
this is Attilio Caravelli. I am an electronic engineer with 20 years of work experience in heterogeneous work environments.
I spent 10 years outside Italy and my last work experience was in Chicago where I played the role of Sr Lab Engineer.
Currently, I am working for Mediatica SPA (Italy) as DevSecOps and Solution Architect.
I would enjoy collaborating with you on this project.
Attilio

**Reply by sciabarracom** [reply]
You welcome Attilio. Attilio is already active and is helping us to build the OpenShift test platform.

**Reply by AmbraDanesin** [reply]
Hi Attilio!
Thank you for this presentation, it's a pleasure 🌟

**Comment by fb-git** [comment]
Hello everyone 👋 
I am Federico and I have joined Michele in this adventure and will provide support in the community.

I have a technical background having worked as sysadmin, system engineer and devops engineer for a number of years before moving into leadership and management.

It's fantastic to see already so many people willing to contribute: first and foremost thanks to each of you!

In the next few days let's organise on the issues and things to do and see how to tackle this beast 💪 
Exciting times!

**Reply by AmbraDanesin** [reply]
Vabbe vabbe a te te conosco quindi bella Fede 😌✌️

**Comment by emanuele-ratti** [comment]
Hi everyone, 
my name is Emanuele and I've been following Michele on LinkedIn for several years: he's really great! This new idea impressed me a lot. I'm just a young banking data analyst (ETL activities only) with a financial background but I hope to learn as much as possible from all of you by following the evolution of this project. 
Thank's!

**Reply by sciabarracom** [reply]
You are welcome Emanuele, let's hope to start soon to discuss what to with everyone.
We are far too busy in investor pitches and other stuff but we will go back soon to code.

**Reply by AmbraDanesin** [reply]
HI Emanuele! I'm happy to read this message, you make me smile :)

**Comment by sderubis-hub** [comment]
Drear all,
my name is Stefano and I wanna collaborate with Michele (that I have known on Linkedin) on this beautiful and interesting project. I'm not a developer but if necessary I've some decades of experience as an AWS Solution Architect, Business Analyst, Project Manager and Senior Data Consultant. 
I can operate also as a Business Plan writer or CFO (2 decades of experience in managing some companies) to guide and analyze some operations. 
Hope to be included in the team and, in the meantime, I wish good luck to everyone for the Project.
Stefano de Rubis

**Reply by sciabarracom** [reply]
Hi stefano given your experience would you cobtribute a terraform that builds ank EKS cluster?

**Reply by AmbraDanesin** [reply]
Hi, Stefano!
I'm pretty sure that you will feel good in this community!

**Comment by sciabarracom** [comment]
Hello all, I just [posted on Linkedin](https://www.linkedin.com/posts/msciab_ciao-a-tutti-volevo-chiedervi-c%C3%A8-qualcuno-activity-6872527909681168384-qUQk) (in italian) a proposal to do some trainings to let people start to help the project. 
Let me know your thoughts.

**Reply by AmbraDanesin** [reply]
And in the end (IT DOESNT EVEN MATTER.........) when will the training start?

**Comment by diegoconti77** [comment]
My name is Diego, and I'm currently working on projects in Manufacturing as a solution architect of MES systems with the following stack: Angular/.Net as Backend.

I'm studying containerization with Docker and K8s and Google Cloud.

I hope to help wherever is possible.

Diego Conti.

**Reply by AmbraDanesin** [reply]
Hi, Diego! Thank you for your message, maybe it will be a hard thing to complete but if we have you and others everything will be easier for sure.
Thanks for join us!

**Comment by capitanfuturo** [comment]
Hi all, my name is Giuseppe Caliendo, and I work as a full-stack developer (Nodejs - Angular). I have been working for several years in Java and now I'm studying docker and Kubernetes.

I want to contribute as much I can. Thanks @sciabarracom for opening this community.

Giuseppe

**Reply by AmbraDanesin** [reply]
It's great, Giuseppe! We sure need someone with your skills... And your nick! Now that we have a "Capitan Futuro" between us things we'll be better ;)

**Comment by AmbraDanesin** [comment]
Actually I forgot to present myself.
I'm Ambra Danesin from Venice (well, near Venice).
What I'm doing here is working with you all, managing the community. So, if you need anything, I'm here to help.

Feel free to contact me here or via chat if you are in Slack!

**Comment by valx** [comment]
Hi all,
I am Valerio, Italian based in London.
I am a data engineer, with some experience with devops.

I see lots of  potential in this project and I am glad to see it moving fast!
I am interested in the trainings (Python for Kubernetes and devops) and potentially to collaborate.

Thanks!

**Reply by AmbraDanesin** [reply]
Hi Valerie from London!
Great to see you here, we sure need all the help! 
See how many people believe in this project is beautiful ❤️

**Comment by tntimo** [comment]
Hello,

My name is Nicola, I've signed up for one of the training (k8s operators) and I'd like to give something back.
I have been working as a software engineer for ~15 years, but it's the first time I try to contribute to an open source project, so I have a lot to learn.
In the last few years I've worked in Python and Go; most recently I've been writing also a lot of Terraform for AWS.
Some time ago I asked Michele on LinkedIn how he does so many things and his answer was to just do it; hence here I am. 

Nicola

**Reply by AmbraDanesin** [reply]
Yeah, that is an answer in Michele's style ahah
Thank you for join us, we too have to learn a lot and this is what binds us together!

**Comment by capoloja** [comment]
Hi everyone, 
I am Pasquale, another "differently young" IT professional, we have been known with Michele and some of you last year. 
I don't really know if I will be able to help with something (test, dockerfiles on RPI, vscode configuration), but I'm sure with you guys there is always fun (and something to learn, of course ;-))

Ciao

**Reply by AmbraDanesin** [reply]
Welcome Pasquale!
Same mood tbh, Michele has a personality that involves everyone ahah
Happy to see you here!

**Comment by Parsiphal** [comment]
Hello everyone, my name is Parsiphal David Coletta. I'm a Software Architect and Health Care Domain Specialist in Dedalus S.p.A. I'm very happy to collaborate on this project and I'm waiting for training.

**Reply by AmbraDanesin** [reply]
Hi, Parsiphal! Welcome on board, we hope you'll find inspiration and fun!
Can I ask you, maybe we know each other on LinkedIn? Cause I remember a guy with the name "David Coletta"!

**Reply by Parsiphal** [reply]
Hello Ambra, thanks for the welcome!
We have a 2nd-degree connection on LinkedIn. Maybe I'm that person! :D

**Reply by AmbraDanesin** [reply]
I've tried to find you but... Are you the one from US or Monaco?
Why there are so many people with your same name that work on this field

**Comment by IT-AlIan** [comment]
Hi guys I'm Alfonso Iannotta and came from Italy so my username is IT-Alian and seems there is a lot of Italy on this project despite our IT industrial development. I started my career with a Michele's java course about 25 years ago, and now I wish to learn again giving an hand on this project. I was a java developer and an application server administrator; now I'm a big data admin

**Reply by AmbraDanesin** [reply]
Hello, Alian! Yes, for now the majority is from Italy but probably we just have to wait ahah
Anyway welcome, happy to see you here!

**Reply by IT-AlIan** [reply]
I'm proud of it, Italians do it better ;) . Anyway I hope this project involve international developers at all

**Reply by AmbraDanesin** [reply]
*Maneskin plays in the background*

**Comment by giusdp** [comment]
Hi everyone, I'm Giuseppe De Palma. I am a soon-to-start PhD student in computer science.

Unfortunately I don't have that much experience unlike many people here, I got into serverless and openwhisk 
for my master thesis work and I'd love to follow and use (and contribute back to) projects like this now.

I'm sure I will learn a lot :)

**Reply by AmbraDanesin** [reply]
Hi, Giuseppe! Even if you are not as experienced as some of our guru don't worry, you will learn!

Thank you for joing us ❤️

**Comment by fcolasante** [comment]
Hi everyone! I'm Francesco Colasante. I just graduated in Engineering in Computer Science at Sapienza.
I have experience in Angular (Typescript and NGRX), Google Cloud Platform (Firebase, Cloud Function in Node.Js, PubSub, and IoT Core), Python
In addition, I have experience in Docker and Docker Swarm. I would like to understand what are the differences with Kubernetes and learn Go.

**Reply by msciabarra** [reply]
You are welcome Francesco. Nuvolaris is a "serverless" layer on top of Kubernetes. It does not look at all like Kubernetes, it looks a lot like Amazon Lambda. Actually it looks like IBM Cloud functions. What we want it to make it available everywhere for everyone running it in Kubernetes clusters.

**Reply by AmbraDanesin** [reply]
Hello Francesco! Thank you you to join us, you are very welcome :)

**Comment by edidome** [comment]
Hello all I am Eugenio, 
I am sixty and I have been programming since 1982 in many languages and different environments but I consider myself a novice in this sector with a great desire to learn.

**Reply by msciabarra** [reply]
Welcome aboard Eugenio. We are going to do some trainings to share knowledge freely.

**Reply by AmbraDanesin** [reply]
Hi Eugenio! Don't worry, here you will learn a lot 😎👌

**Comment by Lukpier** [comment]
Hello everyone!

My name is Luca, I'm a Backend & Data Engineer having 4 years of experience. 
I'm in love with learning continuously new stuff.
So why am I here? because this is definitely the place to do it!

Can't wait to start contributing :)

**Reply by AmbraDanesin** [reply]
Hey, Luca! :)
"Because this is definitely the place to do it!", soooo nice to read it 😌👌

**Reply by msciabarra** [reply]
You welcome Luca, check the list of open issues for inspiration

**Comment by sciabarracom** [comment]
Hi all next week we start with the training! Every tuesday at 18pm we will deliver some trainings on the various technologies of Nuvolaris. The place is this audio/video chat room on Discord: bit.ly/nuvolaris-classroom  do not forget to register to Discord...

**Comment by Telemaco019** [comment]
Hi everyone! My name is Michele, I'm a software engineer developing in Python and Java, with a bit of experience in Go.

In the last year, however, I've been mainly developing Azure infrastructure using Terraform and I've fallen in love with cloud computing and DevOps 😁  I've experienced a bit with Kubernetes and now I would like to learn more about it and about serverless in general. It seems like this is the right place to do that  🙂

Hoping to be of any help, I'm looking forward to contributing!

**Reply by AmbraDanesin** [reply]
Hello Michele! Thank you for your preentation, we look forward to work with you!

**Comment by msciabarra** [comment]
Hello all, tuesday the 25th of January we do another community call, and we want to introduce community members in a video that we record and put on youtube. Just show up, tell who you are and what you do and maybe which kind of contribution you would do in the project.

**Reply by AmbraDanesin** [reply]
And remember guys: 6pm italian time 🥸

**Comment by Corrado-synclab** [comment]
Hi everyone !

I'm Corrado from Ostia, I'm working as sysops in aws since few months. I started study IT couple of years ago, and lately I focused on the DevOps technologies.
So this look like the perfect place to improve my knowledge and get some nice experience, it's even the first time I'm joining a github project so I'm quite excited :)
I would like to contribute, as much as I'm able to. 
I got some experiences with Terraform and Ansible, far from call me an expert on it, but i know enough enough to want to go deeper in it.

Corrado

**Reply by msciabarra** [reply]
Welcome Corrado. Devops stuff is developed currently in the #nuvolaris-operator channel on Discord.

**Reply by AmbraDanesin** [reply]
Hello you! Welcome here, if you need anything you can ask me (well, if it's not too much technical) :)

**Comment by stoiclionroars** [comment]
Hi!
I'm Matteo 'Lion' Martis. I'm a computer sciencentist. In the last three years I worked as java backend developer. 
Mainly I worked with Java (jersey, gwt, spring boot), but I coded also some python scripts and a couple of little 
front-end application with react and vue.
Sometimes I study languages like go, rust, haskell, but I don't know much of them.

I hope I can be helpful without damages.

**Reply by msciabarra** [reply]
I am sure you can. Which parts are you mostly interested?

**Reply by stoiclionroars** [reply]
I mostly interested in  jest testing and go command line because I want to improve my testing skill and I want to deep dive into go development, but I'm open to every helpful collaboration. I didn't think about k8s python operator, because I don't know k8s at all, but I know python; In this hour, I'm watching the video training for the python operator.

**Comment by msciabarra** [comment]
Give a look then to the [first test set](https://github.com/nuvolaris/nuvolaris-testing/tree/main/tests) for runtimes delivered by @giusdp

**Reply by stoiclionroars** [reply]
I looked at it. No problem with javascript and jest. I need to go depth into the application domain.

**Comment by matgerva** [comment]
Hi, 
I am Mattia from Bergamo (Italy). I am working as a software engineer for more than ten years but I have never approached an open source project. I find out the project following Michele on Linkedin and I would like to gradually follow the project as I am completely new at container technology and orchestators, so I have to learn a lot of thing before being able to contribute. I watched videos on youtube channel and I appreciated the open mood and the energy of people involved.

**Comment by Pasquale-Favella** [comment]
Hi, 
i'm Pasquale from Naples (Italy). i have a professional butcher background and it's almost one year trying to code in a professional manner , most of in java but love ardently FE things. 
I've never approached an open source project but fell in love with nuvolaris concept thanks to beautiful posts of michele on Linkedin and videos on Youtube channel.
I don't know of any specific technology used on nuvolaris, but I want to contribute even just with an idea.
Tonight I made a small prototype of a web app in next js that uses github as a cms to document Nuvolaris.

Below is the link to view it:
https://next-nuv-docs.vercel.app/
<img width="947" alt="nuv-docs-dark" src="https://user-images.githubusercontent.com/62385965/158997891-e341bd59-bc03-4f05-b81f-f1937bfccea4.PNG">

even if little, I hope that I can be useful in some way.

Thank you

**Reply by msciabarra** [reply]
THat is wonderful!!! Where we can contact you? Are you on discord? Please message me there...

**Comment by msciabarra** [comment]
> Hi All I am Michele Sciabarrà!
> 
> I am the founder of the Nuvolaris prject.
> 
> I am member of the Apache OpenWhisk project (author of the runtime proxy actionloop used by many runtimes), author of the book Learning Apache OpenWhisk published by O'Reilly, formerly Nimbella Europe Manager and (briefly) Developer Advocate at Digital Ocean who acquired Nimbella.
> 
> This project aims to build an open source, production-ready, distribution of Apache OpenWhisk that installs in every kubernetes, works flawless and it is easy to use with good development tools.
> 
> If you ware interested to collaborate, please get in touch and introduce yourself.

Hi
